SQL/New_patches/2025-02-19_alter_action_issue_tracker_permissions.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE permissions CHANGE `action` `action` enum (
```

Comment:
This will have to be run before the patch above for the patch to succeed (I think you can put them in the same patch)